build/diva/build.sh

```diff
@@ -0,0 +1,6 @@
+echo "---- BUILD Diva ----"
+
+cd diva.js
```

Comment:
I agree with other comments elsewhere.  Having a folder called diva.js seems odd.  I expect file extensions to specify file types.